subscription-example/src/main/resources/templates/admin/disable.html

```diff
@@ -0,0 +1,23 @@
+<html lang="en"
+      xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout}">
+<head>
+  <title th:text="${type}"></title>
+</head>
+<body>
+<div layout:fragment="content" class="status-container">
+  <div class="status">
+    <img th:if="${result == 'success'}" src="/images/thank-you.svg" class="status-image" alt="">
+    <img th:if="${result == 'error'}" src="/images/error.svg" class="status-image" alt="">
+
+    <p class="status-message">
+      <span th:if="${type == 'success' }">Disabled RecurringDetailReference th:text="${recurringDetailReference}"</span>
+      <span th:if="${type == 'error' }">
```

Comment:
Output "${recurringDetailReference}" string on frontend properly if it fails

Reply:
Using correct variable (`result` instead of `type`)